Makefile

```
@@ -107,7 +107,7 @@ slow_tests_diffusers: test_installs
 
 # Run text-generation non-regression tests
 slow_tests_text_generation_example: test_installs
-	python -m pip install triton==3.1.0 autoawq
+	python -m pip install -r requirements_awq.txt
```

```suggestion
	python -m pip install -r examples/text-generation/requirements_awq.txt
```
Makefile is not in the same folder as requirements_awq.txt